### ISSUE TITLE
SUPPORT: Error loading Drawing: Expected double-quoted property name in JSON

### ISSUE THREAD

**Issue by JosephShrekstar** [zsviczian/obsidian-excalidraw-plugin#2113]
### Have you checked the documentation and searched existing issues?

- [X] Yes, I have reviewed the documentation and searched for related issues.

### Your question

Hello!

I Keep getting the error below when trying to open the note as an Excalidraw drawing. 
Does anyone know How to fix this or if its not possible  to be fixed : Any way to prevent this in the future?
Did I do something wrong? 

Thank you!

### Steps you've already tried

I Tried editing the line that is stated, (In the file below and in the image its back to how it was in the beginning) But could still not get it to work.
I'm not sure what exactly is meant by "Expected double-quoted property name" as if I understand it right the property name in that line already seems to be double quoted.
I tried opening it on three different devices (Computer, Laptop, Phone) , all are getting the same error and I tried looking at the sync history and could not find a working one.
I Also tried disabling all other community plugins (Especially  the Incompatible ones) other than Excalidraw. However, I did have MAKE.md , Minimal Theme and Style Settings enabled before this error occured if those could have caused it.

### Expected outcome

Being able to open the file in obsidian as an Excalidraw drawing

### Additional context

The Excalidraw version is the newest (2.6.7).
It is only this file that isn't working as all my other Excalidraw notes work.

The error:
![image](https://github.com/user-attachments/assets/7254da2f-2082-4772-a378-a974a0329bd5)

Position of the error:
![image](https://github.com/user-attachments/assets/6ad55cdf-455e-4fd5-a53e-1fa3ec330942)

The File:
[!!! Formeln Salze Fällung.md](https://github.com/user-attachments/files/17701806/Formeln.Salze.Fallung.md)

**Comment by zsviczian** [comment]
Here's the fixed file: [Formeln.Salze.Fallung.md](https://github.com/user-attachments/files/17712613/Formeln.Salze.Fallung.md)
Based on the file, this issue was caused by Obsidian sync, or whatever sync service you are using, mixing two versions of the file at the middle of the file.
1704311 is the position from the start of the JSON, not the start of the MD file.

Anyone reading this in the future here are the steps I took to restore the file:
- opened the file in Notepad++, cut the top of the MD file, everything until the opening `{` character of the JSON.
- navigated to the indicated position and saw what is in the file below.
- I deleted the end of the file and restored the JSON to have a clean close (see second image)
- Because it still didn't load, and I wasn't able to find the error, I opened Obsidian developer console and coped the entire json from the first `{` to the last `}` and initialized a variable in the console with the value hoping the console will give me a proper error message to find the source of the issue. Instead, the console loaded the data, so then I clicked copy object to clipboard and pasted it back to the file, and it opened.

![image](https://github.com/user-attachments/assets/92bf20ca-8c5f-4174-b587-7ea0ef2c2c75)

![image](https://github.com/user-attachments/assets/cf8afb07-de47-4920-a794-f78e22668756)
